test/server-unit/util.spec.js

```diff
@@ -63,4 +64,42 @@ describe('util.js', () => {
     const result = util.renameKeys(a, keyMap);
     expect(result).to.deep.equal([]);
   });
+
+  it('#formatCsvToJson should return a json from a csv file', () => {
+    /**
+     * The structure of the sample csv file (ohada-accounts.csv)
+     * =========================================================
+     * "account_number",  "account_label",    "account_type", "account_parent"
```

Comment:
Can we call this column (and the others) "account number" instead of "account_number"?  It would be easier for our users to type.